requirements.txt

```diff
@@ -0,0 +1,2 @@
+-r ./requirements/defaults.txt
+-r ./requirements/dev.txt
```

Comment:
I can understand if this is a requirement of Heroku, but we're no longer maintaining these files.  Is there a solution were you could be using poetry?  I'd hate for our work with dependabot and keeping up with security updates to be over looked.

Reply:
Resolving outdated commit.